tests/update_eve_image/testdata/update_eve_image_oci.txt

```diff
@@ -1,5 +1,5 @@
 # Default EVE version to update
-{{$eve_ver := "9.6.0"}}
+{{$eve_ver := "12.0.1"}}
```

Comment:
Can we use different versions for oci and http tests if it is possible? I wrote few words about the reason [here](https://github.com/lf-edge/eden/pull/967#discussion_r1572855910).

Reply:
We will have to wait for another 12.X release. Currently there are only two (12.0.0 is broken) and we need one as the base version.

Reply:
OK, converted to draft.

Reply:
@milan-zededa @uncleDecart We do have a 12.1.0 and 12.2.1 which can be used for this if you just want different versions.
(It would be nice to get less unneeded failures in the eden runs.)